packages/nameguard-sdk/src/index.ts

```diff
@@ -490,7 +519,7 @@ export class NameGuard {
   }
 
   /**
-   * Inspects a single name with NameGuard. Provides a `NameGuardReport` including:
+   * Inspects a single name with NameGuard. If the `name` is uninspected returns `UninspectedNameGuardReport`; else returns a `InspectedNameGuardReport` including:
```

Comment:
@djstrong I think it will be nice to update this to more formally refer to the new `inspected` field rather than the abstract idea of being inspected / uninspected.

Reply:
I don't have idea how.